fuel-graph-node/graph/src/runtime/mod.rs

```diff
@@ -392,6 +392,22 @@ pub enum IndexForAscTypeId {
     FuelStorageSlotArray = 5420,
     FuelBytesArray = 5421,
     FuelU64Array = 5422,
+    FuelReceipt = 5423,
+    FuelReceiptArray = 5424,
```

Comment:
Looks like you apply the same change to multiple enums. Maybe we could import the other enum instead?

Reply:
I think we have to do like this here and on the .ts side.